### PR TITLE
Add ability to edit container config

### PR DESCRIPTION
**Pull request by davemcorwin**
- Edit container configuration in admin
- Fix notification/alert components so it disappears automatically
- Remove users list from sites page (up for debate)
- Add admin site fetch/update routes
- Update site serializer

### CODE REVIEW

File: admin-client/src/components/Alert.svelte [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<script>
```
Comment (davemcorwin) [inline admin-client/src/components/Alert.svelte:1]: Moved this to it's own component so the `onMount` functionality actually fires correctly!

File: admin-client/src/components/SiteMetadata.svelte [hunk 2]
```diff
@@ -3,7 +3,6 @@
   import ExternalLink from './ExternalLink.svelte';
```
Comment (davemcorwin) [inline admin-client/src/components/SiteMetadata.svelte:3]: Sooo, i removed this bc I wasn't including the users in the query and it was causing problems. Trying to think of why we would want/need to see the users...

File: api/models/site.js [hunk 3]
```diff
@@ -149,6 +149,24 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false,
     },
+    basicAuth: {
```
Comment (davemcorwin) [inline api/models/site.js:152]: apparently the `getterMethods` section is deprecated. Also, using setters allows the other code to interact with the site model in a more natural way, without needing to worry about these fields actually stored in the `config` field.

Comment (apburnes) [reply api/models/site.js:152]: 👍 👍 The virtual data type looks a lot cleaner and definitely makes interacting with `config` more natural.

File: api/serializers/site.js [hunk 4]
```diff
@@ -1,47 +1,74 @@
...
+const viewLinks = {
+  demoViewLink: 'demo',
+  previewLink: 'preview',
+  viewLink: 'site',
+};
 
-  delete object.site_users__user_sites;
-  delete object.config; // may contain sensitive info ie: basicAuth
+// Eventually replace `serialize`
+function serializeNew(site, isAdmin = false) {
```
Comment (apburnes) [inline api/serializers/site.js:40]: For future context, maybe we're more semantic with `isAdmin` ie (`isPlatformAdmin`, `isFederalistAdmin`) so we're not to be confused with the repository admin.

Comment (apburnes) [reply api/serializers/site.js:40]: Though `isAdmin` might be obsolete soon if we let folks set container config. 🤷

Comment (davemcorwin) [reply api/serializers/site.js:40]: updating to `isSystemAdmin`

**Comment by davemcorwin** [comment]
@amirbey If we want to test `siteToItem` then we should add a test for it, but that's not really what this particular test is trying to achieve. Looking at this now, a better approach may have been to stub out `siteToItem`. Either way, can we handle that improvement in an additional PR?

**Comment by amirbey** [comment]
@davemcorwin - that's fine. it's already checked in the SiteToItem test so your right ... no need to verify here

**Pull request merged**